packages/graphql/src/queries/lists.graphql

```diff
@@ -37,3 +38,142 @@ query GetListItems($predicateId: numeric, $objectId: numeric) {
     }
   }
 }
+
+query GetListDetails(
+  $globalWhere: triples_bool_exp
+  $userWhere: triples_bool_exp
+  $tagPredicateId: numeric
+) {
+  globalTriplesAggregate: triples_aggregate(where: $globalWhere) {
+    aggregate {
+      count
+    }
+  }
+  globalTriples: triples(where: $globalWhere) {
+    label
+    id
+    vaultId
+    counterVaultId
+    subject {
+      id
+      vaultId
+      label
+      walletId
+      image
+      type
+      tags: asSubject_aggregate(
+        where: { predicateId: { _eq: $tagPredicateId } }
+      ) {
+        nodes {
+          object {
+            label
+            vaultId
+            taggedIdentities: asObject_aggregate {
```

Comment:
We likely want to split this out -- we can use the `asObject_aggregate` to get the `aggregate` field and then use the `asObject` to access the data. This isn't too critical here since we're not doing pagination at this level and we can review these again once we do a full pass of all our queries/fragments.

Nice job adding the tags to the query!